Sources/Perception/PerceptionRegistrar.swift

```diff
@@ -207,24 +206,13 @@ extension PerceptionRegistrar: Hashable {
 
   var isInSwiftUIBody: Bool {
     for callStackSymbol in Thread.callStackSymbols {
+      let mangledSymbol = callStackSymbol.utf8
+        .drop(while: { $0 != .init(ascii: "$") })
+        .prefix(while: { $0 != .init(ascii: " ") })
```

Comment:
I was able to get a small performance boost by doing all the string manipulation in UTF8 world instead of doing string splitting.